pac4j-core/src/test/java/org/pac4j/core/engine/DefaultLogoutLogicTests.java

```diff
@@ -110,7 +110,7 @@ public void testLogoutPerformed() {
         profiles.put(NAME, new CommonProfile());
         addProfilesToContext();
         call();
-        assertEquals(200, context.getResponseStatus());
+        assertEquals(204, context.getResponseStatus());
         assertEquals("", context.getResponseContent());
```

Comment:
Theoretically, an empty body is not the same than no body (null content if you want to talk in programming language terms). Not sure if this is important or not though :)

Reply:
Does it change something at the HTTP level (between empty and null)?

Reply:
With an empty content, the response can have a content-type as well as a content-length (of 0), while with no body, there should be none of the two… so if you do set an empty body, you have the risk of generating a response with this incoherent meaning…
Also it may depends on the lib that handles the response generation, maybe they won't include the body at all in case there is a 204 status.

Ok, I realise I'm nitpicking, let's not bother about that :)

Reply:
OK. So you're in line to merge it "as is"?